src/rad/mainframe.cpp

```diff
@@ -968,6 +968,11 @@ void MainFrame::UpdateFrame()
 	wxMenu* menuEdit = GetMenuBar()->GetMenu( GetMenuBar()->FindMenu( wxT( "Edit" ) ) );
 	wxToolBar* toolbar = GetToolBar();
 
+	toolbar->EnableTool(ID_SAVE_PRJ, AppData()->IsModified());
+	wxMenu* pmenuFile = GetMenuBar()->GetMenu(GetMenuBar()->FindMenu(_("File")));
```

Comment:
No type encoding in the name, we only use m_, s_ and maybe g_ from the hungarian notation, remove the p here please